src/plugins/kibana_react/kibana.json

```diff
@@ -3,5 +3,6 @@
   "version": "kibana",
   "ui": true,
   "server": false,
+  "extraPublicDirs": ["common"],
```

Comment:
According to the docs this attribute is deprecated. Should we re-export the values from `kibana_react/public` and `kibana_react/server` so the import conventions are maintained?

Reply:
APM imports this into its own `common` directory, so from where would we import it if we were doing that?

Reply:
That's a good question. Maybe we should keep it this way if the bundler doesn't offer a better way.